topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin_jetbrains_runtime.md

```diff
@@ -0,0 +1,126 @@
+
+<include from="tools_intellij_platform_gradle_plugin.md" element-id="Beta_Status"/>
+<include from="tools_intellij_platform_gradle_plugin.md" element-id="faq"/>
+
+The IntelliJ Platform Gradle Plugin requires JetBrains Runtime (JBR) for running IDE locally, testing, or performing other actions that require Java Runtime adjusted by JetBrains.
+
+For the best experience, it is recommended to rely on JetBrains Runtime bundled with the IntelliJ Platform used for development.
+If one is not available, i.e., when targeting IntelliJ Platform snapshot or nightly releases, there are other ways available for providing JetBrains Runtime in a required version or variant.
+
+## Bundled (default, recommended)
+
+IDE releases provided with JetBrains Download CDN are OS-specific and contain JetBrains Runtime (JBR) already bundled within the archive.
+This is the recommended way for developing plugin, as it comes in exactly the same version as JetBrains IDEs delivered to users.
+
+To use this approach, no extra steps are required but declare the IntelliJ Platform, see: [](tools_intellij_platform_gradle_plugin_dependencies_extension.md#target-platforms).
+
+## Obtained with IntelliJ Platform from Maven
+
+If the IntelliJ Platform is resolved with the IntelliJ Maven Repository instead of JetBrains Download CDN, it has no JetBrains Runtime (JBR) bundled.
+In such a case, it is necessary to provide it in the suitable version with the version read from the <path>[intellijPlatform]/dependencies.txt</path> file.
+
+This can be easily achieved by using the `jetbrainsRuntime()` repository helper pointing to [JetBrains Runtime GitHub Releases](https://github.com/JetBrains/JetBrainsRuntime/releases/) and dependency helper with no arguments provided:
+
+```kotlin
+repositories {
+  mavenCentral()
+  intellijPlatform {
+    defaultRepositories()
+    jetbrainsRuntime()
+    // ...
+  }
+}
+
+dependencies {
+  intellijPlatform {
+    intellijIdeaCommunity("%ijPlatform%")
+
+    jetbrainsRuntime()
+
+    // ...
+  }
+}
+```
+
+## Declared explicitly
+
+It is possible to explicitly specify JetBrains Runtime version, variant, or exact build with:
+
+```kotlin
+repositories {
+  mavenCentral()
+  intellijPlatform {
+    defaultRepositories()
+    jetbrainsRuntime()
+    // ...
+  }
+}
+
+dependencies {
+  intellijPlatform {
+    intellijIdeaCommunity("%ijPlatform%")
+
+    jetbrainsRuntime(version, variant, architecture)
+    // or
+    jetbrainsRuntimeExplicit(explicitVersion)
+
+    // ...
+  }
+}
+```
+
+Provided `version`, `variant`, and `architecture` parameters along with the `explicitVersion` are used to resolve the JetBrains Runtime archives published on [GitHub Releases](https://github.com/JetBrains/JetBrainsRuntime/releases/) page.
+
+To correctly understand the pattern, refer to the archive names in format `jbr_[prefix]-[jdk]-[os]-[arch]-b[build].tar.gz`, such as `jbr_jcef-21.0.3-osx-aarch64-b446.1.tar.gz`, which can be split into the following parts:
```

Comment:
````suggestion
To correctly understand the pattern, refer to the archive names in format:
```
jbr_[prefix]-[jdk]-[os]-[arch]-b[build].tar.gz
```
e.g.:

```
jbr_jcef-21.0.3-osx-aarch64-b446.1.tar.gz
```
which can be split into the following parts:
````
Please avoid breaking the line when explaining the pattern.